04 - Guides, Workflows, & Courses/Guides/Using Discord.md

```diff
@@ -0,0 +1,49 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+---
+
+# Using Discord
+
+## Don't ask to ask, just ask
```

Comment:
A little introduction of how/why this is needed? Right off the bat these two things seem unrelated.

Reply:
It might also be best at the end of the file, once we have explained some of the other Discord stuff?

Reply:
good point, did both